src/OrchardCore/OrchardCore.ContentManagement.Abstractions/Metadata/Models/ContentTypeDefinitionOptions.cs

```diff
@@ -18,4 +18,9 @@ public class ContentTypeDefinitionOptions
     /// In this dictionary, the 'key' denotes the content type, while the 'value' corresponds to the driver options.
     /// </summary>
     public Dictionary<string, ContentTypeDefinitionDriverOptions> ContentTypes { get; } = [];
+
+    /// <summary>
+    /// Configure the default driver options.
+    /// </summary>
+    public ContentTypeDefinitionDriverOptions Default { get; set; } = new();
```

Comment:
@lampersky please keep this in the driver private.

Reply:
but this way, we also have possibility to change default behavior globally, maybe somebody wants to configure it in a different way than all set to true, what do you think?

Reply:
Not really. This should be kept private. We don't want anyone outside to change it. The only way to change it is to use the Configure method to configure the options

Reply:
> The only way to change it is to use the Configure method to configure the options

but it is still configurable via `Configure<>()`, so with my change, you could go with code like this:

```
services.Configure<ContentTypeDefinitionOptions>(options => {

    // configure default behavior
    options.Default = new ContentTypeDefinitionDriverOptions
    {
        ShowCreatable = false,
        ShowListable = true,
        ShowDraftable = false,
        ShowVersionable = false,
    }
    
    // configure behavior for specific content types...
    options.ContentTypes.TryAdd("ContentType1", new ContentTypeDefinitionDriverOptions
    {
        ShowCreatable = true,
        ShowListable = false,
        ShowDraftable = true,
        ShowVersionable = true,
    });

    // configure another content type
    options.ContentTypes.TryAdd("ContentType2", new ContentTypeDefinitionDriverOptions
    {
        ShowCreatable = false,
        ShowListable = false,
        ShowDraftable = true,
        ShowVersionable = true,
    });
});
```

I still think this could be useful. If somebody wants to simply hide all those options for every possible content types, he/she can just configure default without re-configuring all existing ContentTypes.

```
services.Configure<ContentTypeDefinitionOptions>(options => {
    // configure default behavior, it will be applied when there is no other configuration provided
    options.Default = new ContentTypeDefinitionDriverOptions
    {
        ShowCreatable = false,
        ShowListable = true,
        ShowDraftable = false,
        ShowVersionable = false,
    }
});
```

maybe it would be good to ask what others think?

Reply:
What if you have multiple projects and each one is modifying this global variable? This is not a good option to expose publicly. There is a reason why I made it private when I introduced this option. But, feel free to ping others and get more feedback.

Reply:
@lampersky thank you. Once you make that change ping me again and I'll merge it.

On the other side, if there is a good case where we need to support this, you can create a separate PR and add more options to handle such a case using ContentTypeDefinitionOptions instead.

For example, add options like 

ListableByDefault
CreatableByDefault
....

And set these all to true by default. These would then be used to set the private default options in the driver. But, I would keep that change as part of a separate PR to see if it is something we would take separately. The problem with such approach is that the UI needs these functions. If you set ListableByDefault to false, then any content type that is created by the UI will not be visible on the UI even if you created it by UI.

Reply:
@lampersky Are you able to make the last updates soon? I would like to merge this soon so it fixes the issue for others. If you like I can hijack the PR and make the last change and then merge it.

Reply:
I will do it today, I have it on my TODO list.